packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -2024,14 +2024,12 @@ void main() {
     expect(inkWell.highlightColor, expectedHighlightColor);
   });
 
-  testWidgets('ExpandIcon.disabledColor uses expandIconColor color when canTapOnHeader is true', (WidgetTester tester) async {
```

Comment:
Is there a reason why this test for the disabled color is being removed?

Reply:
@Piinks That test was for the original fix for this issue that broke the mouse pointer. I removed that fix, and fixed it a different way, so the old test is redundant.